_rules/html-page-title-descriptive-c4a8a4.md

```diff
@@ -37,18 +37,15 @@ htmlHintIgnore:
 
 ## Applicability
 
-This rule applies to the first [HTML][] `title` element that
-
-- is a [descendant](https://dom.spec.whatwg.org/#concept-tree-descendant) of the `html` element of a [web page](https://www.w3.org/TR/WCAG21/#dfn-web-page-s), and
-- contains [children](https://dom.spec.whatwg.org/#concept-tree-child) that are [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
+This rule applies to the first [document title][] of each [html web page][] that contains [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
```

Comment:
Probably fine to remove the word "first", since there can only be one document title.

```suggestion
This rule applies to the [document title][] of each [html web page][] that contains [text nodes](https://dom.spec.whatwg.org/#text) that are not only [whitespace](#whitespace).
```